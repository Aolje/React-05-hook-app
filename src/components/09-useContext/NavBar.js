import React from 'react';
import { Link } from 'react-router-dom';

export const NavBar = () => {
  return (
    <nav className="navbar navbar-expand-sm navbar-dark bg-dark">
      {/* <div class="container-fluid">
        <Link className="navbar-brand" path='/'>useContext</Link>
        <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
          <div className='navbar-nav'> */}
      <ul>
        <li>  <Link className="nav-item active" to='/'>Home</Link></li>
        <li>  <Link className="nav-item active" to='/about'>About</Link></li>
        <li>  <Link className="nav-item active" to='/login'>Login</Link></li>
      </ul>
      {/* </div>
        </div>
      </div> */}
    </nav>
  )
}
