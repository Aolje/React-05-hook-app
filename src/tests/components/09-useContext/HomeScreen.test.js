import React from 'react';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Pruebas en <homeScreen />', () => {
    const user = {
        name: 'alejandro',
        email: 'alejo@alejo.com'
    }
    const wrapper = mount(
        <UserContext.Provider value={user}>
            <HomeScreen />
        </UserContext.Provider>
    )

    test('Debe de mostrar correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })
})