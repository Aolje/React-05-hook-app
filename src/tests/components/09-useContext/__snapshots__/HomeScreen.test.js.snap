// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <homeScreen /> Debe de mostrar correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre />
</div>
`;
